pkg/ddl/reorg.go

```diff
@@ -73,8 +73,7 @@ type reorgCtx struct {
 	references atomicutil.Int32
 }
 
-// newContext gets a context. It is only used for adding column in reorganization state.
-func newContext(store kv.Storage) sessionctx.Context {
+func newReorgSessCtx(store kv.Storage) sessionctx.Context {
```

Comment:
Can we add a comment？ Before it was only used for adding columns，will it be used in both modifying column and the add index?

Reply:
it has many usage but they all belong to write reorganization, so I rename it `newReorgSessCtx` to indicate reorg.